docker/candlepin-base/setup-supervisord.sh

```diff
@@ -2,6 +2,8 @@
 
```

Comment:
Yes; the deploy script and the test setup in cp-test relies on several services to be started/running:  tomcat, qpid, sshd (nice to have, not strictly needed for unit test running).  The problem is that in a container systemd itself hasn't started.  This leads us down two paths:

1)  We go ahead and start it the same way you would on a normal machine startup using `/usr/sbin/init` ;  this however requires a 'super' privileged mode like this: `docker run -it -e "container=docker" --privileged=true -d  --security-opt seccomp:unconfined --cap-add=SYS_ADMIN -v /sys/fs/cgroup:/sys/fs/cgroup:ro --name $NAME $IMAGE bash -c "/usr/sbin/init"` .  This only complicates our nice new simplified docker-compose setup, and introduces all the fun problems you get by running docker in  privileged mode.

2)  We could manually call the daemons and fork in our deploy script, which would require a little more logic, but supervisord already does this for us.  The question here being should we re-implement things or find a better tool.

I found myself going down a bit of a rabbit hole with the above so I decided limit the scope of this PR to its original goals and punt on that work for later.